.internal-ci/helm/consensus-node-config/templates/node-ledger-distribution-secret.yaml

```diff
@@ -12,6 +12,6 @@ stringData:
   AWS_SECRET_ACCESS_KEY: {{ .awsSecretAccessKey | quote }}
   AWS_REGION: {{ .awsRegion | quote }}
   LEDGER_DISTRIBUTION_S3_BUCKET: {{ .s3Bucket | quote }}
-  LEDGER_DISTRIBUTION_START_FROM: {{ .startFrom | quote }}
-  LEDGER_DISTRIBUTION_S3_PATH: {{ tpl .awsPath $ | quote }}
+  MC_START_FROM: {{ .startFrom | quote }}
```

Comment:
pull this out so we don't get confused, the ledger-distribution wrapper sets this